scout/server/blueprints/alignviewers/templates/alignviewers/igv_sashimi_viewer.html

```diff
@@ -84,6 +80,20 @@ <h3>{{case}}: Splice junctions</h3>
                                 }
                             ]
                           }, // end of sashimi track with data
+                          { // genes track
+                            name: geneTrack.name,
+                            type: geneTrack.type,
+                            format: geneTrack.forma,
```

Comment:
```suggestion
                            format: geneTrack.format,
```

Reply:
Thanks! Weird that the tracks were displayed anyway. The param is not probably required, but let's have it, and correct :)